src/frontend/src/common/components/library/data_table/style.ts

```diff
@@ -73,3 +84,20 @@ export const HeaderCheckbox = styled(Checkbox)`
     `;
   }}
 `;
+
+export interface AlignProps extends Props {
+  align?: string;
+}
+
+export const TableHeader = styled("div", {
+  shouldForwardProp: (prop) => !doNotForwardProps.includes(prop as string),
```

Comment:
do you still need this line? it looks like TableHeader doesn't have a `header` prop?